src/rtc/rtc_s1_single_job.py

```diff
@@ -260,7 +260,8 @@ def compute_layover_shadow_mask(radar_grid: isce3.product.RadarGridParameters,
                                 extraiter_rdr2geo: int=10,
                                 lines_per_block_rdr2geo: int=1000,
                                 threshold_geo2rdr: float=1.0e-7,
-                                numiter_geo2rdr: int=25):
+                                numiter_geo2rdr: int=25,
+                                memory_mode: str=''):
```

Comment:
Does it run if `memory_mode` is not set by the user?

Reply:
Good catch. I'll change the default value to `None`